app/Http/Controllers/Api/AirlineController.php

```diff
@@ -29,6 +34,8 @@ public function store(StoreAirlineRequest $request): JsonResponse
     public function update(UpdateAirlineRequest $request, Airline $airline): JsonResponse
     {
         $airline->update($request->validated());
+        $existingCities = City::whereIn('id', explode(',', $request['cityIds']))->get();
+        $airline->cities()->syncWithoutDetaching($existingCities);
```

Comment:
```suggestion
        $cityIds = explode(',', $request->string('cityIds')->toString());

        $airline->cities()->syncWithoutDetaching($cityIds);
```
Same thing here. `syncWithoutDetaching()` makes sense here, because the airline exists from before, so you'd want to make sure you're keeping the previously attached cities.
Also, don't forget about spacing between lines.